### ISSUE TITLE
Add file creation dates to API

### ISSUE THREAD

**Issue by LucaScorpion** [Tocabi/clearcloud#37]
Currently we only expose the last modified date, not the creation date. While getting the creation date does seem be be very different for the different platforms (mostly Unix/Windows), it should be possible.

**Comment by ChappIO** [comment]
Yeah I left it out for now because in my testing it was dependent on the OS + FileSystem combination. Didn't want to delay 1.0 for this.
